### DISCUSSION TITLE
Rendered Mode (like Intellij)

### DISCUSSION THREAD

**Discussion by xvbcfj** [zed-industries/zed#24003] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

https://www.jetbrains.com/guide/java/tips/toggle-rendered-view/

Jetbrains IDEs support a mode that displays comments similar to how they appear during the documentation hover.
Similar functionality would be very useful.

**Comment by JosephTLyons** [comment]
This is a neat feature! Inline rendering of doc comments!

**Comment by pleahmacaka** [comment]
It will be supercool

**Comment by nkcodec** [comment]
it would be great to have this toggle view feature

**Comment by xvbcfj** [comment]
any updates? or mentoring instructions so that somebody outside of the zed team can pick this up.
